mrscake/src/App.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
why you use this useeffect, usestate? I just want to know functionality.

Reply:
this gonna be applied to know who(role) is logged in, so to adopt website for his functionality